azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -99,59 +107,82 @@ def test_auth_provider_creation(self, mocker, client_class, connection_string, c
     @pytest.mark.parametrize(
         "ca_cert",
         [
-            pytest.param(None, id="No CA certificate"),
+            pytest.param(None, id=" No CA certificate "),
             pytest.param("some-certificate", id="With CA certificate"),
         ],
     )
+    @pytest.mark.parametrize(
```

Comment:
Also, with the way these parameters are stacking, we may need to at some point reconsider how these tests are structured now that there's much more variable input.

But don't worry about this now.

Reply:
yeah we'll have to think about how to improve these tests without making them an ultimate labyrinth....